src/components/JobConfiguration.vue

```diff
@@ -62,6 +62,12 @@
         <ion-button size="small" fill="outline" @click="saveChanges()">{{ $t("Save changes") }}</ion-button>
       </div>
     </div>
+
+    <div class=" actions mobile-only">
+      <ion-button expand="block" fill="outline" color="medium" @click="skipJob">{{ $t("Skip once") }}</ion-button>
```

Comment:
I think we should add same condition for both condition.

Reply:
I have added conditions to buttons in the mobile view also.